resources/js/Pages/community-adapters.jsx

```diff
@@ -73,10 +73,13 @@ export default function () {
         <Li>
           <A href="https://github.com/j0ack/flask-inertia">Flask</A>
         </Li>
+        <Li>
+          <A href="https://github.com/tobimori/kirby-inertia">Kirby CMS</A>
+        </Li>
       </Ul>
       <P>
         If you have an adapter you'd like listed here, please send a{' '}
-        <A href="https://github.com/inertiajs/inertiajs.com/edit/master/resources/js/Pages/community-adapters.js">
+        <A href="https://github.com/inertiajs/inertiajs.com/blob/master/resources/js/Pages/community-adapters.jsx">
```

Comment:
I think this is actually the URL we want here 👍 

```suggestion
        <A href="https://github.com/inertiajs/inertiajs.com/edit/master/resources/js/Pages/community-adapters.jsx">
```